server/src/utils/gptServiceHelper.ts

```diff
@@ -1,3 +1,68 @@
+import { QueriedProduct } from '../../../shared/types';
```

Comment:
maybe a better name for the file would just be prompts?
gptServiceHelper is a little confusing haha